core/src/main/scripts/importer/validateData.py

```diff
@@ -2898,3 +2898,4 @@ def main_validate(args):
                 1: 'failed',
                 2: 'not performed as problems occurred',
                 3: 'succeeded with warnings'}.get(exit_status, 'unknown')))
+    sys.exit(exit_status)
```

Comment:
`metaImport.py` imports and calls the function directly. The code above is only run when `validateData.py` is run as a standalone script.